tasks/configure.yml

```diff
@@ -1,16 +1,16 @@
 ---
 
 - name: protect my.cnf
-  file: path='{{mysql_hardening_mysql_conf}}' mode=0600 owner=root group=root follow=yes
+  file: path='{{mysql_hardening_mysql_conf_file}}' mode=0600 owner=root group=root follow=yes
```

Comment:
About second commit in this PR: On untouched installations of MariaDB/MySQL the permissions of /etc/my.cnf include read for others, that's why mysql process can read config files. However, when we restrict it to 0600 root:root, mysql process can't read it. This should be fixed, but what would be the best way? 0460 mysql:root, 0640 root:mysql or even 0400 mysql:root would do, since root can read it in any case.